connections/sim/sim_connection.py

```diff
@@ -170,6 +171,14 @@ def _handleDigitalPinWrite(self):
         self._hw_sim.digital_write(pin, value)
         LOGGER.info(f"SIM: Pin {pin} set to {value} (device_address={self.device_address})")
 
+    def _handlePinMode(self):
+        length = self._getLength()
+        assert length == 2
+        pin, mode = self.stdout.read(2)
+
+        self._hw_sim.set_pin_mode(pin, mode)
+        LOGGER.info(f"SIM: Pin {pin} set to {mode} (device_address={self.device_address})")
```

Comment:
This log is identical to the one in `_handleDigitalPinWrite()` we should somehow differentiate them to prevent confusion in the future.

Reply:
Adjusted this now as well